### ISSUE TITLE
Question - Desktop App Window Size 

### ISSUE THREAD

**Issue by thothloki** [flexxui/flexx#643]
Is there a way for me to resize the desktop app window when i run the app? Currently, it is the "standard" size when I run it, but the amount of components I have on it, the size could easily be half.

I am making a basic app to just sit in a corner of a screen, but would like it as small as possible.

I might have missed it in the docs.

**Comment by almarklein** [comment]
That should be the `size` param of the `launch()` function/method. It's indeed a bit buried, because Flexx' version of `launch` calls webruntime's version: https://webruntime.readthedocs.io/en/latest/

**Comment by thothloki** [comment]
Thank you. I will dig into that.

**Comment by thothloki** [comment]
i guess i am just confused how to use the DesktopRuntime's launch_app feature
`if __name__ == '__main__':
    app = DesktopRuntime.launch_app(CryptoCalc, title='Crypto Calc',size=(100,200))
    app.run()`
this is what i have but am getting the following errors
`Assigning result of a function call, where the function has no return`
`Unexpected keyword argument 'title' in unbound method call`
`Unexpected keyword argument 'size' in unbound method call`
`No value for argument 'url' in unbound method call`

**Comment by thothloki** [comment]
right now i am using
`
if __name__ == '__main__':
    app.launch(CryptoCalc, runtime='app or chrome-app', title='Crypto Calc')
    app.run()
`
so far there is no way for me to set the window size

**Comment by almarklein** [comment]
You can add it there:
```py
app.launch(CryptoCalc, runtime='app or chrome-app', title='Crypto Calc', size=(100, 200))
                                                           here --------/
```

**Comment by rwang0417** [comment]
Hello,

I also run into this issue. The title name I set is shown correctly, but the window size is still the default.
Here is the code:
```
a = flx.App(TuneAI_Demo, title='TuneAI Demo', size=(1200, 100))
a.launch('app', size=(1200, 100))  
flx.run()
```
Also, I tried the demo twente.py from the reference. The size parameter there is not working either.

I run the code from the command line on Windows 10. The Python version is 3.8.3. Any thought about what I might have missed?
